qutip/tests/core/test_superoper.py

```diff
@@ -65,7 +65,7 @@ def testOperatorVector(self):
         N = 3
         rho1 = qutip.rand_dm(N)
         rho2 = qutip.vector_to_operator(qutip.operator_to_vector(rho1))
-        assert (rho1 - rho2).norm('max') < 1e-8
+        np.testing.assert_allclose(rho1.full(), rho2.full(), 1e-8)
```

Comment:
It would have been better to keep these changes out of this unrelated PR.  This stuff is being fixed on `master` already in a relevant PR, and it's going to cause merge conflicts now.  I can deal with it, though, so it's probably not worth the effort of removing.

Reply:
PS: I'd like to merge up `master` to `dev.major` again once your outstanding PRs are in.  Ideally we'll do that before the GSoC students start doing any real work, so the docs are up.